encord/orm/dataset.py

```diff
@@ -192,6 +192,32 @@ def from_dict(cls, json_dict: Dict) -> Dataset:
         )
 
 
+@dataclasses.dataclass(frozen=True)
+class TitleDataHashMapping(Formatter):
+    data_hash: str
+    title: str
+
+    def from_dict(self, json_dict: Dict) -> TitleDataHashMapping:
+        return TitleDataHashMapping(json_dict["data_hash"], json_dict["title"])
+
+
+@dataclasses.dataclass(frozen=True)
+class AddPrivateDataResponse(Formatter):
+    """Response of add_private_data_to_dataset"""
+
+    imported_data_rows: List[str]
+    title_id_mapping: List[TitleDataHashMapping]
+    imported_videos: List[str]
```

Comment:
can we remove this?
since this is not needed and used in the SDK?

Reply:
They aren't used in the SDK but they do get returned from it.

Reply:
Lets remove this field, this will confuse users of the SDK